src/components/services/object_fit_polyfill/readme.md

```diff
@@ -0,0 +1,10 @@
+# Object fit Polyfill
+
+We use `object-fit` to create the create large responsive background images within things like the promo block. Older browsers (unsurprisingly) don't like it so we have a slightly hacky polyfill for it.
+
+On the parent element that uses `object-fit: cover` also add the class `js-objectFit_container`. If the browser doesn't support `object-fit` (checked via Modernizr) then the image will be included as a background image instead.
```

Comment:
Class needs updating to js-object-fit-container. Maybe also clarify "on the parent **of the** element..."